HOW_TO_RUN_SAMPLES_WITH_SPRING_BOOT_3.md

```diff
@@ -0,0 +1,28 @@
+# How to run samples with Spring Boot 3?
+
+The current examples are built based on Spring Cloud Azure 4.x by default. If you want to use Spring Cloud Azure 6.x based on Spring Boot 3, this document will tell you how to do it.
+
+## How to make a sample support running with Spring Boot 3?
+
+1. It's required to add a document *HOW_TO_RUN_THIS_SAMPLE_WITH_SPRING_BOOT_3.md* to introduce the differences, it should be put together with the sample's *README.md*. The outline and contents of the document are as follows:
+
+```markdown
+# How to run this sample with Spring Boot 3?
+
+1. Update parent pom path
+
+2. Update dependencies(optional)
+
+3. Update Java code(optional)
+
+```
+
+2. Update the parent pom file to *azure-spring-boot-samples-6.x-pom.xml*.
```

Comment:
1. The order is not right. It should not be `2` here.
2. Not only update the parent file path, the artifact id should be udpated, too.
3. The content of `Update parent pom path` should be fixed. How about writing the update parent here? Ah.. the relative path is not fixed. I think we can add a comment here.

How about moving these contents info a new file named `HOW_TO_RUN_THIS_SAMPLE_WITH_SPRING_BOOT_3_TEMPLATE.md` besides current file?